docs/zz_generated.kubermaticConfiguration.ce.yaml

```diff
@@ -557,14 +557,17 @@ spec:
       - v1.28.9
       - v1.28.11
       - v1.28.12
+      - v1.28.14
       - v1.29.0
       - v1.29.1
       - v1.29.2
       - v1.29.4
       - v1.29.6
       - v1.29.7
+      - v1.29.9
       - v1.30.3
-      - v1.31.0
+      - v1.30.5
+      - v1.31.1
```

Comment:
More a question than request for changes:

For KKP v2.25, the [highest semver we support](https://github.com/kubermatic/kubermatic/blob/release/v2.25/docs/zz_generated.kubermaticConfiguration.ee.yaml#L546-L559) are the following:
- v1.27.13
- v1.28.9
- v1.29.4

Shouldn't we simply get rid of v1.29.6 and v1.29.7 for v1.29 and just keep v1.29.9 in the list? Same for v1.30; get rid of v1.30.3 and just keep v1.30.5. For v1.28, we can get rid of v1.28.11, v1.28.12.

Reply:
Thanks, I'll update the PR.

Reply:
PR is updated. I also included EKS/AKS updates.